drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -235,7 +240,11 @@ def service_form_definition(service_type_id:, project_id:)
       service_type = Hmis::Hud::CustomServiceType.find_by(id: service_type_id)
       raise HmisErrors::ApiError, 'Service type not found' unless service_type.present?
 
-      Hmis::Form::Definition.find_definition_for_service_type(service_type, project: project)
+      definition = Hmis::Form::Definition.find_definition_for_service_type(service_type, project: project)
+
+      # Similar to record_form_definition above, we always want to return a definition if we possibly can, so use the
+      # default HUD Service form for HUD service types. For Custom service types, return empty because we can't determine which form to use.
+      definition || (service_type.hud_service? ? Hmis::Form::Definition.with_role(:SERVICE).where(managed_in_version_control: true).first : nil)
```

Comment:
I think the `||` with a ternary is a but more complicated than it needs to be. What about this (untested)?
```suggestion
      definition ||= Hmis::Form::Definition.with_role(:SERVICE).where(managed_in_version_control: true).first if service_type.hud_service?
      definition
```

Reply:
Yup this seems to work well with local testing, but I added a new spec too so I'll wait for the tests to pass before merging. Thanks for the suggestion!